lib/matplotlib/figure.py

```diff
@@ -174,6 +174,7 @@ def __init__(self, **kwargs):
         # see self.align_xlabels and self.align_ylabels and
         # axis._get_tick_boxes_siblings
         self._align_label_groups = {"x": cbook.Grouper(), "y": cbook.Grouper()}
+        self._align_title_groups = cbook.Grouper()
 
```

Comment:
```suggestion
        self._align_label_groups = {"x": cbook.Grouper(), "y": cbook.Grouper(), "title": cbook.Grouper}
```

Does it make sense just to add another grouper here to stop having another property on the method?